react/src/components/organisms/FooterSlim/index.js

```diff
@@ -16,60 +16,77 @@ import IconPhone from 'MayflowerReactBase/Icon/IconPhone';
 // eslint-disable-next-line import/no-unresolved
 import IconLaptop from 'MayflowerReactBase/Icon/IconLaptop';
 
-const FooterSlim = (props) => (
-  <footer className="ma__footer-slim" id="footer">
-    <div className="ma__footer-slim__container ma__container">
-      {props.siteLogo}
-      <div className="ma__footer-slim__container__inner">
-        <div className="ma__footer-slim__info">
-          <div className="ma__footer-slim__title">{props.title }</div>
-          <p>{props.description }</p>
-          <p className="ma__footer-slim__copyright">
-            &copy;
-            {' '}
-            {moment().year()}
-            {' '}
-            Commonwealth of Massachusetts
-          </p>
+const FooterSlim = ({
+  title, description, siteLogo, stackedLogo, links, contact
```

Comment:
`stackedLogo` needs a default value.

Reply:
Also `description`. Maybe:
```suggestion
  title = '', description = '', siteLogo, stackedLogo = false, links, contact
```

Reply:
I will add a default value to stackedLogo. but `title`, `description` are required props, so they won't need default props?

Reply:
That makes sense. 👍